src/visitors/visitor_utils.cpp

```diff
@@ -215,4 +227,28 @@ std::string to_json(const ast::Ast& node, bool compact, bool expand, bool add_nm
     return stream.str();
 }
 
+std::pair<std::string, std::unordered_set<std::string>> statement_dependencies(
+    const std::shared_ptr<ast::Expression>& lhs,
+    const std::shared_ptr<ast::Expression>& rhs) {
+    std::string key;
+    std::unordered_set<std::string> out;
+
+    if (!lhs->is_var_name()) {
+        return {key, out};
+    }
+
+    key = to_nmodl(lhs);
+    key.erase(std::remove(key.begin(), key.end(), '\''),
```

Comment:
There are 2 cases:

a' -> a
F'[0] -> F[0] 

I tried with "to_nmodl" but I get the '. I tried with getting only the var_name but I get F only for the second (not F[0]). Do you know of something better?

Reply:
basically I have 2 cases that I need to take care:
a' -> a
F'[0] -> F[0]

if I get the VAR_NAME node I get only F (instead of F[0]) if i use to_nmodl I get a' instead of a. Do you have a better routine?

Reply:
OK, after having also discussed in chat. the `'` operator could be also parsed as an operator and treated as such, which would probably clean up such manual string manipulation and make the semantics a bit stronger. But, this is not the scope of this PR, so we leave it out. Could you please create an issue though? Then we can decide later whether we want to implement this or not.

Reply:
added here: https://github.com/BlueBrain/nmodl/issues/511